### PR TITLE
Bugfix, setting properly X-Frame-Options

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys, we resolve:
-We set suitable X-Frame-Options header
Closes: #2791

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Check the max buckets by default in healthcheck and increase them [#2901](https://github.com/wazuh/wazuh-kibana-app/pull/2901)
 - Added a prompt wraning in role mapping if run_as is false or he is not allowed to use it by API [#2876](https://github.com/wazuh/wazuh-kibana-app/pull/2876)
+- Added a suitable X-Frame-Options header [#2791](https://github.com/wazuh/wazuh-kibana-app/issues/2791)
```
Comment (Desvelao) [inline CHANGELOG.md:11]: This change should is in:
```
## Wazuh v4.1.1 - Kibana 7.10.0 , 7.10.2 - Revision 4102
```

File: server/plugin.ts [hunk 2]
```diff
@@ -66,6 +66,14 @@ export class WazuhPlugin implements Plugin<WazuhPluginSetup, WazuhPluginStart> {
...
 
   public async setup(core: CoreSetup, plugins: PluginSetup) {
+
+    core.http.registerOnPreResponse((request, response, toolkit) => {
+      const additionalHeaders = {
+        'x-frame-options': 'sameorigin',
+      };
+      return toolkit.next({ headers: additionalHeaders });
+    })
+
```
Comment (Desvelao) [inline server/plugin.ts:76]: Maybe you could move this code below.

**Comment by Desvelao** [comment]
I tested the PR and the responses include the header `x-frame-options` to `sameorigin`. This includes the statics files too.
image

**Comment by sortizowlh** [comment]
Network header response tested. Now every response has `x-frame-options` header

**Review by sortizowlh — APPROVED** [review]
Check!

**Review by pablomarga — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (CPAlejandro) [reply CHANGELOG.md]: I have change it, thanks you!

**Pull request merged**